lesson02/assignment/css/_components.content.scss

```diff
@@ -1,17 +1,84 @@
-html {
-    box-sizing: border-box;
+/* media queries to make responsive for different screens */
+@media screen and (max-width: 576px) {
+        .main {
+            grid-template-columns: 1fr;
+            margin: .5rem;
+            padding: .5rem;
+            border: solid $gray 1rem;
+        }
+        .box2 {
+          display: grid;
+          grid-column-start: 1;
+          grid-column-end: 2;
+        }
 }
-
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
+@media screen and (min-width: 576px) {
+        .main {
+            grid-template-columns: 1fr 2fr;
+            margin: 40px;
+            padding: 30px;
+            border: solid $gray 40px;
```

Comment:
For responsivity, we should be using relative units (rem, %) for almost everything. That allows for the maximum flexibility with the way users interact with the site (zoom, setting large font sizes, etc).